src/fixes.rs

```diff
@@ -19,7 +19,9 @@ trait Fix {
     fn fix_warnings(&self, warning_lines: &[usize], lines: &mut Vec<LineEntry>) -> Option<usize> {
         let mut count: usize = 0;
         for line_number in warning_lines {
-            let line = lines.get_mut(line_number - 1)?;
+            let line = lines
+                .iter_mut()
+                .find(|entry| entry.number == *line_number)?;
```

Comment:
Why did you decide to rewrite this code?

Reply:
Because line number != index.
Multiline values contain multiple line numbers in one.

And this led to a strong dependence of the order of fixes (e.g, ordering)